src/main/groovy/io/openliberty/tools/gradle/utils/LooseWarApplication.groovy

```diff
@@ -2,10 +2,16 @@ package io.openliberty.tools.gradle.utils;
 
 import io.openliberty.tools.common.plugins.config.LooseApplication
 import io.openliberty.tools.common.plugins.config.LooseConfigData
+import org.apache.commons.io.FilenameUtils
+import org.gradle.api.Project
 import org.gradle.api.Task
+import org.gradle.api.artifacts.Dependency
+import org.gradle.api.artifacts.ProjectDependency
 import org.gradle.api.logging.Logger
 import org.gradle.api.tasks.bundling.War
 
+import java.nio.file.Path
+
```

Comment:
You should undo the changes to this file.

Reply:
This had 2 util methods which moved to DevTaskHelper. I have done an optimze import on this

Reply:
but these new imports are not used by this file

Reply:
corrected